docs/source/markdown/podman-pod-create.1.md

```diff
@@ -120,6 +120,14 @@ Add a DNS alias for the container. When the container is joined to a CNI network
 
 Disable creation of /etc/hosts for the pod.
 
+#### **--pid**=*pid*
```

Comment:
I was thinking "process ID (pid) when reading this.  I'll let others over-rule me, but I'd rather the option be named something else, in case we ever manipulate the pid later down the road.  Perhaps `--pid-mode`, regardless, at least  change the variable:
```suggestion
#### **--pid**=*pid-mode*
```

Reply:
I feel like `--pid-ns` might be more accurate if anything @TomSweeneyRedHat. Only thing is, [container create](http://docs.podman.io/en/latest/markdown/podman-create.1.html#pid-pid) calls it `--pid`

Reply:
Ugh.  They should be the same in both places, and it looks like Docker has gone with `--pid` too, so we probably ought to stay with `--pid`.  Rather than `*pid-mode*` at line 123, I'd be fine with `*pid-ns*` if you prefer for the variable

Reply:
okay @TomSweeneyRedHat I am good with keeping it as `--pid`, probably best so people don't get confused as to what this flag is compared to container create and docker